rest/admin_api_test.go

```diff
@@ -5055,3 +5055,53 @@ func TestResyncPersistence(t *testing.T) {
 	fmt.Printf("RT2 Resync Status: %s\n", resp2.BodyBytes())
 	assert.Equal(t, resp.BodyBytes(), resp2.BodyBytes())
 }
+
+// Repros CBG-2416
+func TestDBReplicationStatsTeardown(t *testing.T) {
```

Comment:
I'd like to avoid putting any more tests in `[admin_]api_test.go` where possible

There's already a set of REST-API based ISGR tests in `replication_api_test.go` that seems like a more appropriate and focused place for this test